src/Dashboard/ApiControllers/DiagnosticsController.cs

```diff
@@ -1,117 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Web.Http;
```

Comment:
So you're deleting all this code? Why not instead just remove it from the sln and leave in this repo? Until we decide to move it out to a different repo (if we do).